Cargo.toml

```diff
@@ -89,6 +89,8 @@ members = [
     "language/move-prover/bytecode",
     "language/move-prover/docgen",
     "language/move-prover/errmapgen",
+    "language/move-prover/interpreter",
+    "language/move-prover/interpreter-testsuite",
```

Comment:
Why in its own crate?

Reply:
To avoid cyclic dependency.

`move-unit-test` depends on `move-prover/interpreter`, therefore, in `move-prover/interpreter`, we cannot depend on `move-unit-test` to run the tests

Reply:
In fact if crate A depends on crate B, then still the B testsuite can depend on A. Rust breaks the cycle for tests. We have examples for this in the prover, e.g. docgen depends on the prover main crate, and docgen tests depend on the prover crate. It's not that important though.